test/jdk/java/net/httpclient/HttpServerAdapters.java

```diff
@@ -105,6 +106,16 @@ public static abstract class HttpTestRequestHeaders {
         public abstract Set<Map.Entry<String, List<String>>> entrySet();
         public abstract List<String> get(String name);
         public abstract boolean containsKey(String name);
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (!(o instanceof HttpTestRequestHeaders headers)) return false;
+            return Objects.equals(entrySet(), headers.entrySet());
```

Comment:
This code confused me a bit initially. Each of the subclasses of `HttpTestRequestHeaders` has a `headers` private member and I misread this line to be comparing the headers entrySet with itself. Would it be better if we renamed this local variable (in the instanceof line above) to `otherHeaders` to avoid any confusion?

Reply:
Why doesn't this comment appears when I click on the "Files Changed" tab? Strange.
Anyway - yes - good point - will change `headers` to `other` - I agree it's confusing.